.github/workflows/release.yml

```
@@ -0,0 +1,108 @@
+
+      - name: Move DEB
+        run: mv *.deb dist/
+
+      # Build RPM package
+      - name: Build RPM
+        uses: jiro4989/build-rpm-action@v2
+        with:
+          spec_file: dkms/mt76-mt7925-dkms.spec
+
+      - name: Move RPM
+        run: mv *.rpm dist/ 2>/dev/null || true
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2035:info:1:4: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)